tests/callbacks/test_run_directory_uploader.py

```diff
@@ -15,6 +15,11 @@
 @pytest.mark.parametrize("use_procs", [False, True])
 @pytest.mark.timeout(15)
 def test_run_directory_uploader(tmpdir: pathlib.Path, use_procs: bool, dummy_state: State, dummy_logger: Logger):
+    try:
+        import libcloud
+        del libcloud
```

Comment:
```suggestion
        del libcloud
```

Reply:
The `del` helps avoid unused variable warnings in the IDE